auction-server/src/opportunity/entities/opportunity_svm.rs

```diff
@@ -62,6 +67,9 @@ pub struct OpportunityCreateSvm {
     pub slot:               Slot,
 }
 
+// Opportunity can be refreshed after 10 seconds
+const MIN_REFRESH_TIME: Duration = Duration::seconds(10);
```

Comment:
this means that we may send the same opportunity every 10 seconds to the searchers right? If so, maybe increase it.